app/registries/foreman/plugin.rb

```diff
@@ -316,7 +316,7 @@ def extend_template_helpers(*mods)
     # Add Compute resource
     def compute_resource(provider)
       return if @compute_resources.include?(provider)
-      if !provider.is_a?(Class) || !(provider < ComputeResource)
+      unless provider.is_a?(Class) && provider < ComputeResource
```

Comment:
I'm against this, unless with && or || makes it very hard for me to read the condition. I prefer to use if in case of composed conditions